system/security/securesocket/securesocket.cpp

```diff
@@ -1631,6 +1855,18 @@ SECURESOCKET_API ISecureSocketContext* createSecureSocketContextSecret(const cha
         return createSecureSocketContext(sockettype);
 }
 
+SECURESOCKET_API ISecureSocketContext* createSecureSocketContextSecretSrv(const char *mtlsSecretName)
+{
+    if (!queryMtls())
+        throw MakeStringException(-100, "TLS secure communication requested but not configured");
```

Comment:
trivial: makeStringException or makeStringExceptionV if has vargs preferred.

Reply:
#define MakeStringException makeStringExceptionV

Reply:
I just used what was used in the same file in other places.

Reply:
I think MakeStringException is considered deprecated. 
I believe explicit varieties (makeStringException/makeStringExceptionV) were introduced because it was unsafe that MakeStringException assumed vargs and with non-constant strings unintentionally cause it to read args of the stack.

@ghalliday - might be able to confirm thinking.

Reply:
my policy is:
a) any new code should not use MakeStringException
b) change existing MakeStringException's in existing methods/functions if the PR makes any non-trivial change to those methods/functions.

I wouldn't go through a whole existing file and change all.

Reply:
ok, thx. I changed the two new uses I added to be makeStringExceptionV()